### ISSUE TITLE
Volume Booster

### ISSUE THREAD

**Issue by wyrl** [superpoweredSDK/Low-Latency-Android-iOS-Linux-Windows-tvOS-macOS-Interactive-Audio-Platform#459]
I'm looking for volume booster to create hearing aid app...

**Comment by superpoweredSDK** [comment]
How is this an issue with the Superpowered SDK?

**Comment by wyrl** [comment]
I have programmed with "AndroidAudioAdvanceIO" that recording mic while playing sound, but the volume is very low...

**Comment by superpoweredSDK** [comment]
What is "AndroidAudioAdvanceIO"?

**Comment by wyrl** [comment]
Sorry for my mistake. I mean "SuperpoweredAndroidAudioIO". I tried to use "SuperpoweredAndroidAudioIO" it works for me. Unfotunately, volume is too low.
Example:
`SuperpoweredAndroidAudioIO audioIO = new SuperpoweredAndroidAudioIO(samplerate, buffersize, true, true, audioProcessing, NULL, SL_ANDROID_RECORDING_PRESET_CAMCORDER, SL_ANDROID_STREAM_MEDIA, 1);`

Then I tried to put "SuperpoweredVolume" in "audioProcessing" method but it doesn't works.
```
static bool audioProcessing(void * __unused clientdata, short int *audioInputOutput, int numberOfSamples, int __unused samplerate) {
    SuperpoweredVolume((float) audioInputOutput, (float) audioInputOutput, 1.0, 4.0, numberOfSamples);
    return true;
}
```
Sorry for my bad english :)
